### DISCUSSION TITLE
Swaync replacing KDE notification service

### DISCUSSION THREAD

**Discussion by giomxrcode** [ErikReider/SwayNotificationCenter#478] [Q&A]
Hello, guys!

I use KDE and Hyprland on the side and I'm using swaync as a notification service for Hyprland, but I've noticed that swaync is replacing Plasma's notification service on KDE.

So, I was wondering (and been looking around for answers, but no luck), is there a way to stop swaync from replacing Plasma's notification service on KDE?

(Yes, I already know how to start it up on Hyprland, so I already covered that!)

**Comment by ErikReider** [comment]
`systemctl --user disable --now swaync.service` should work :)

**Reply by theguynamedcj** [reply]
Hi im a bit late but anytime i receive a notification it calls swaync.
